dashboard/src2/pages/LoginSignup.vue

```diff
@@ -187,8 +247,37 @@ export default {
 					product: this.$route.query.product,
 					new_signup_flow: true
 				},
+				onSuccess(account_request) {
+					this.account_request = account_request;
+					this.accountRequestCreated = true;
+				}
+			};
+		},
+		verifyOTP() {
+			return {
+				url: 'press.api.account.verify_otp',
+				params: {
+					account_request: this.account_request,
+					otp: this.otp
+				},
+				onSuccess(key) {
+					window.open(`/dashboard/setup-account/${key}`, '_self');
+				},
+				onError() {
+					// Already the error messagee is shown in UI <ErrorMessage />
+					// So, no need to show the default error toast
+				}
```

Comment:
removed the fallback error handler since it's showing duplicate errors https://github.com/frappe/press/commit/c43e4d3e8a3d492688e08a40ced306c47a5d0f9e